pypi_formula_mappings.json

```diff
@@ -44,6 +44,7 @@
   },
   "buku": {
     "package_name": "buku[server]",
+    "extra_packages": ["flask==1.1.4"],
```

Comment:
Flask 1.1.3 introduced upper bounds for some dependencies (for compatibility reasons).

The Python dependency resolution sees two paths - use an older Flask (1.1.2) and allow other dependencies to use the latest version, or use 1.1.4 but limit other dependencies.

It seems it picked the former, which is not what we want as it's broken. This is a hint to make it not do that.

Reply:
👍